data-prepper-plugins/cloudwatch-logs/src/main/java/org/opensearch/dataprepper/plugins/sink/cloudwatch_logs/config/CloudWatchLogsSinkConfig.java

```diff
@@ -49,6 +62,11 @@ public class CloudWatchLogsSinkConfig {
     @Max(50)
     private int workers = DEFAULT_NUM_WORKERS;
 
+    @JsonProperty("custom_headers")
```

Comment:
For STS, we use a configuration named `sts_header_overrides`. We should stay consistent. So perhaps `header_overrides` would be a better name.